src/js/routes.coffee

```diff
@@ -0,0 +1,2 @@
+module.exports = (match) ->
+    match '', 'Home#index'
```

Comment:
Not knowing the syntax, I feel like this is weird, is it not meant to be '/'? Or is blank just the same as root?